showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+---
+title: SARS-CoV-2 sequencing in Estonia 2021-2022
+contributors: [Diana Pilvar, Hedi Peterson]
+description: Full-genomic sequencing and analysis of SARS-CoV-2 strains in Estonia during 2021-2022.
+affiliations: [University of Tartu, Republic of Estonia Health Board, EE]
+related_pages:
+  showcase: [COVID19GalaxyProject]
+page_id: korogenoest 
+---
+
+## Introduction 
+
+Sequencing initiatives in Estonia began in Spring 2020. The KoroGeno-EST-3 and KoroGeno-EST-2022 projects took over the sequencing activities in May 2021, and their work concluded at the end of 2022. Here, we describe the workflow of samples and data employed by the [KoroGeno-EST-3 and KoroGeno-EST-2022 projects](https://kliinilinemeditsiin.ut.ee/et/sisu/eesti-sars-cov-2-taisgenoomide-jarjestamine-ja-analuus-korogeno-est-1est-2est-3) .
+
+## Who is the KoroGeno-Est showcase intended for?
+
+This showcase provides a high-level overview of the full workflow that was used to monitor the SARS-CoV-2 situation in Estonia by sequencing a proportion of positive PCR samples in the country. The showcase could be beneficial for a general overview but also trigger potential contact with research groups or health boards that need to set up their automatic data handling and sequencing pipelines.
+
+
+{% include image.html file="/KoroGenoEst_graphicalabstract.jpeg" caption="The figure illustrates major analysis steps carried out in Estonia to sequence and analyse SARS-CoV-2 PCR positive samples" alt="In Estonia the SARS-CoV-2 sequencing was carried out by University of Tartu in collaboration with the SYNLAB Eesti and Health Board Estonia. The Synlab provided information on the basic sample metadata and PCR outcome, Health Board provided sample metadata on e.g. age, gender, county, vaccination information. The central data storage was at GitLab instance at the University of Tartu. The University of Tartu research groups merged the data, created plate information files for sequencing, collecting the sequencing results, generated custom reports, interactive dashboards and brokered data to ENA. In addition to data sequenced in Estonia, also data from ECDC was analysed in similar manner to provide accurate overview of the situation." %}
+
+
+In Estonia the SARS-CoV-2 sequencing was carried out in collaboration with the SYNLAB Eesti, Health Board Estonia and University of Tartu. The Synlab provided information on the basic sample metadata and PCR outcome, Health Board provided sample metadata on e.g. age, gender, county, vaccination information. 
+
+The central data storage was at GitLab instance at the University of Tartu. The University of Tartu research groups merged the data, created plate information files for sequencing, collecting the sequencing results, generated custom reports, interactive dashboards and brokered data to ENA. In addition to data sequenced in Estonia, also data from ECDC was analysed in similar manner to provide accurate overview of the situation.
```

Comment:
```suggestion
The central data storage was at a GitLab instance at the University of Tartu. The University of Tartu research groups merged the data, created information files per plate for sequencing, collected the sequencing results, generated custom reports, interactive dashboards and brokered data to European Nucleotide Archive ([ENA](https://www.ebi.ac.uk/ena/browser/home)). In addition to data sequenced in Estonia, also data from ECDC was analysed in a similar manner to provide accurate overview of the situation.
```

> also data from ECDC was analysed in a similar manner to provide accurate overview of the situation
What is the geographical coverage of the used data from ECDC? Can it be added to the text?

Reply:
Specified the ECDC sample info